core/storage/statistics/gae_models.py

```diff
@@ -1134,6 +1138,128 @@ def delete_playthroughs_multi(cls, playthrough_ids):
         cls.delete_multi(instances)
 
 
+class LearnerAnswerDetailsModel(base_models.BaseModel):
+    """Model for storing the answer details that a learner enters when they
+    are asked for an explanation of their answer. Currently, the model supports
+    exploration states and questions. One instance of this model is created for
+    each of these objects.
+    The id of this model will be the hash which is automatically generated when
```

Comment:
No. See how other classes specify the ID and do something similar here. It is really important to specify the generation mechanism clearly.

Reply:
Done